src/utils/isPlainObject.js

```diff
@@ -1,6 +1,6 @@
 /**
- * @param {any} obj The object to inspect.
- * @returns {boolean} True if the argument appears to be a plain object.
+ * @param {*} obj The object to inspect.
```

Comment:
I don't mind changing `any` to `*` but then this needs to be done everywhere (including API reference doc).
I'm also happy with keeping it called `any` :-)